osu.Game/Screens/Edit/Setup/MetadataSection.cs

```diff
@@ -28,25 +28,25 @@ private void load()
                 },
                 artistTextBox = new LabelledTextBox
                 {
-                    Label = "Artist",
```

Comment:
It feels kinda awkward without this though.

![image](https://user-images.githubusercontent.com/1329837/111434902-76c34780-8743-11eb-90b7-7e519f1161c1.png)

Reply:
Can you not see the placeholder text?

Reply:
The placeholder text is only shown if there's nothing there.

Here's an example where the values are not so silly:
![image](https://user-images.githubusercontent.com/1329837/111435186-cbff5900-8743-11eb-8333-b4bc2097b9d8.png)

Reply:
Right. But that's no worse than what it was before, right? I can try and add a label if you think it's best to do so as part of this PR.

Reply:
Oh, I removed the label. I see.